sentry_sdk/profiler.py

```diff
@@ -289,9 +317,9 @@ def slice_profile(self, start_ns, stop_ns):
                         frames[frame] = len(frames)
                         frames_list.append(
                             {
-                                "name": frame[0],
-                                "file": frame[1],
-                                "line": frame[2],
+                                "name": frame.name,
+                                "file": frame.file,
+                                "line": frame.line,
```

Comment:
If you don't mind using `function`/`filename`/`lineno`/`colno`, that'd be great. We renamed them to align with the `Frame` object in the protocol but still provide compatibility though.

Also, we added `colno` in case you want/can report the column.

Reply:
I have this in a follow up 👍